tests/unit/modules/test_virt.py

```diff
@@ -1946,6 +1946,51 @@ def test_update(self):
         with self.assertRaises(SaltInvocationError):
             virt.update("my_vm", boot={"efi": "Not a boolean value"})
 
+        # Update memtune parameter case
+        memtune = {
+            "soft_limit": "0.5g",
+            "hard_limit": "1024",
+            "swap_hard_limit": "2048m",
+            "min_guarantee": "1 g",
+        }
+
+        invalid_number = {
+            "soft_limit": "3.4.MB",
+        }
+
+        invalid_unit = {"soft_limit": "2HB"}
```

Comment:
Please move the test data definition next to the test itself: it helps visualizing where the tests are starting